.eslintrc.js

```diff
@@ -1,28 +1,28 @@
 module.exports = {
   env: {
     browser: true,
-    commonjs: true,
-    es2021: true
+    es2021: true,
   },
-  extends: ['plugin:react/recommended', 'airbnb', 'prettier', 'plugin:jsdoc/recommended'],
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+  ],
+  overrides: [
+  ],
   parserOptions: {
-    ecmaFeatures: {
-      jsx: true
-    },
-    ecmaVersion: 14
+    ecmaVersion: 'latest',
+    sourceType: 'module',
   },
-  plugins: ['prettier', 'jsdoc'],
+  plugins: [
+    'react',
+  ],
   rules: {
-    'react/function-component-definition': ['error', { namedComponents: 'arrow-function' }],
-    'react/jsx-props-no-spreading': 'off',
-    'react/no-unescaped-entities': 'off',
-    'react/prop-types': 'off',
-    'no-console': 'off',
-    'jsx-a11y/label-has-associated-control': ['error', { assert: 'either' }],
-    'arrow-body-style': ['error', 'as-needed'],
-    'no-param-reassign': ['error', { props: false }],
-    'jsdoc/valid-types': 'off',
-    'no-alert': 'off',
-    'import/no-extraneous-dependencies': 'off'
-  }
+    'react/prop-types': 0,
+    'react/jsx-props-no-spreading': 0,
+    'jsx-a11y/label-has-associated-control': 0,
+    'no-param-reassign': 0,
+    'no-alert': 0,
+    'comma-dangle': 0,
+    'react/function-component-definition': 0
```

Comment:
This requires all react function components to follow a specific style. It also disallows arrow functions. Airbnb does this because the names of arrow functions are inferred, which can make debugging more difficult. But that's not a common opinion, and not one I think is worth enforcing.